src/main/java/com/denizenscript/depenizen/bukkit/objects/mythicmobs/MythicSpawnerTag.java

```diff
@@ -0,0 +1,341 @@
+            this.spawner = spawner;
+        }
+        else {
+            Debug.echoError("Mythic Spawner referenced is null!");
+        }
+    }
+
+    public MythicSpawner getSpawner() {
+        return spawner;
+    }
+
+    private String prefix;
+    MythicSpawner spawner;
+
+    @Override
+    public ObjectTag setPrefix(String prefix) {
+        this.prefix = prefix;
+        return this;
+    }
+
+    @Override
+    public ObjectTag getObjectAttribute(Attribute attribute) {
+        return tagProcessor.getObjectAttribute(this, attribute);
+    }
+
+    @Override
+    public String getPrefix() {
+        return prefix;
+    }
+
+    @Override
+    public String debug() {
+        return prefix + "='<A>" + identify() + "<G>' ";
+    }
+
+    @Override
+    public boolean isUnique() {
+        return true;
+    }
+
+    @Override
+    public String getObjectType() {
+        return "MythicSpawner";
+    }
+
+    @Override
+    public String identify() {
+        return "mythicspawner@" + spawner.getName();
+    }
+
+    @Override
+    public String identifySimple() {
+        return identify();
+    }
+
+    @Override
+    public String toString() {
+        return identify();
+    }
+
+    public static void registerTag(String name, TagRunnable.ObjectInterface<MythicSpawnerTag> runnable, String... variants) {
+        tagProcessor.registerTag(name, runnable, variants);
+    }
+    public static ObjectTagProcessor<MythicSpawnerTag> tagProcessor = new ObjectTagProcessor<>();
+
+    public static void registerTags() {
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.name>
+        // @returns ElementTag
+        // @plugin Depenizen, MythicMobs
+        // @description
+        // Returns the name of the MythicSpawner.
+        // -->
+        registerTag("name", (attribute, object) -> {
+            return new ElementTag(object.getSpawner().getName());
+        });
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.cooldown>
+        // @returns DurationTag
+        // @plugin Depenizen, MythicMobs
+        // @mechanism MythicSpawnerTag.cooldown
+        // @description
+        // Returns an ElementTag(Number) of the MythicSpawner's cooldown.
+        // -->
+        registerTag("cooldown", (attribute, object) -> {
+            return new DurationTag(object.getSpawner().getCooldownSeconds());
+        });
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.mythic_mobs>
+        // @returns ListTag(MythicMobsMobTag)
+        // @plugin Depenizen, MythicMobs
+        // @description
+        // Returns a ListTag of all active MythicMobs from this spawner.
+        // -->
+        registerTag("mythic_mobs", (attribute, object) -> {
+            ArrayList<MythicMobsMobTag> list = new ArrayList<>();
+            for (UUID uuid : object.getSpawner().getAssociatedMobs()){
+                list.add(MythicMobsMobTag.valueOf(uuid.toString()));
+            }
+            return new ListTag(list);
```

Comment:
Just start with a ListTag and `addObject` on it, don't randomly stick an ArrayList in the middle